docs/DeveloperGuide.md

```diff
@@ -6,33 +6,183 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
+The sequence diagrams for our main class, CantVasMain and its interactions with
+other classes.
+![img.png](img.png)
+![img_1.png](img_1.png)
 
 ## Product scope
 ### Target user profile
-
-{Describe the target user profile}
+NUS Students.
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+Users can use this app to keep track of their expenditures and therefore they can better manage their finances.
+Students can use this app to calculate their GPA grade
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...                 | I want to ...                                 | So that I can ...                                    |
+|---------|--------------------------|-----------------------------------------------|------------------------------------------------------|
+| v1.0    | student                  | focus on financial responsibility             | view a breakdown of his expenditures by category     |
+| v1.0    | Proactive business major | calculate my course grades on CantVas         | see which mod I have to SU.                          |
+| v1.0    | careless user            | delete wrong items in the expenditure tracker | add back the correct item in the expenditure tracker |
 
 ## Non-Functional Requirements
 
 {Give non-functional requirements}
 
+## Environment Setup
+Java JDK 8 or above
+An IDE that supports Java (e.g., IntelliJ IDEA, Eclipse)
+
 ## Glossary
 
 * *glossary item* - Definition
 
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+# Motivational messages
+The Motivational module prints a randomly generated motivational quote for the user whenever the program starts 
+
+# Expenditure
+The expenditure module is comprehensive allows users to key in their expenses to enable them to store all their 
+
+## Overview
+The expenditure module allows users to key in their expenses to enable them to store all their 
+expenses.
+This guide will walk developers through how to use the expenditure function to track their expenses
+
+### List of Operations:
+#### Deleting an expenditure:
+Deleting an expenditure in numerical form, by referencing its index in the array.
+Format: e/ del/ INDEX
+Example: e/ del/ 3
+#### Adding an expenditure:
+Adding an expenditure in numerical form.
+Format: add n/DOLLARS_AND_CENTS d/DD.MM.YYYY
```

Comment:
Similarly for this, for the code format, it might be better to represent the format as "add n/[DOLLARS_AND_CENTS] d/[DD.MM.YYYY]" to make it clearer to the user which are the parameters to change.